benchmarks/tensorexpr/rnn_eltwise.py

```diff
@@ -65,3 +65,41 @@ def default_configs():
         return [[64, 512]]
 
 benchmark.register_benchmark_class(RNNEltwise)
+
```

Comment:
I actually had a quick question about the graph in this benchmark case.
https://github.com/pytorch/pytorch/blob/4c5ef9b13b59f62aeae37111b51620e5d2672f3b/benchmarks/tensorexpr/rnn_eltwise.py#L32-L45

The graph seems to assume that the GEMM's are done before entering in this graph and somewhere else the GEMMs of all four LSTM gates are computed together in a fused GEMM op, which leads to the chunk operator. Is this correct? I just wanted to make sure if this models the situation of running a real LSTM.

Reply:
Yeah, so short answer here is I'm not sure if this is doing the correct thing.  The intent here was basically to take the elementwise portion of the fastrnns `jit` lstm cell (https://github.com/pytorch/pytorch/blob/master/benchmarks/fastrnns/cells.py#L25) and isolate it into this test harness.  At the time we were chasing down some regressions in the tensorexpr fuser and the guess was it was related to fused kernel dispatch in the TE dispatcher.   Anyways, honestly, I'm not 100% sure that I did the extraction correctly, and even if I did it may or may not reflect performance of the real lstm cell.  So, ymmv.  If you spot something obviously wrong I'd happily accept a fix :-)